projects/storefrontapp/src/app/spartacus/features/organization/organization-registration-wrapper.module.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule, Type } from '@angular/core';
+import { CDCB2BRegisterModule } from '@spartacus/cdc/organization/user-registration';
+import { OrganizationUserRegistrationModule } from '@spartacus/organization/user-registration';
+
+import { environment } from '../../../../environments/environment';
+
+const extensions: Type<any>[] = [];
+
+if (environment.cdc) {
+  extensions.push(CDCB2BRegisterModule);
+}
+
+@NgModule({
+  imports: [OrganizationUserRegistrationModule, ...extensions],
+})
+export class OrganizationRegistrationWrapperModule {}
```

Comment:
What do you think about aligning the name of the wrapper module with the orignal module?
OrganizationUserRegistrationModule -> OrganizationUserRegistrationWrapperModule?